### ISSUE TITLE
Stuck in system

### ISSUE THREAD

**Issue by DalynWoodard** [endless-sky/endless-sky#6041]
I've been playing Endless Sky for a while and love it. This past week I had some computer issues and ended up  having to redownload the game, of course getting the latest update. Something new happened. I got a notification that "If you take off now you will be launching with a ship that cannot leave system." If I stay on planet there's nothing to do to change it, and taking off, sure enough can't leave the system. Is there something to keep from triggering this and is there some way to get around it/fix it so I can leave the system or is the ship I've been working on for over a year essentially dead in the water?

**Comment by Zitchas** [comment]
That error is most often cause by a lack of hyperdrive or jump drive.

1. What ship are you using?
2. What system are you in?
3. What version of Endless Sky are you running?

**Comment by DalynWoodard** [comment]
I am using a Falcon wrship and jumped to the system no problem. I don't see how I could have lost the hyperdrive since I wasn't disabled or boarded, but I'll load it up later and check. Unfortunately I can't remember if the system I'm in has an outfitter. I'm in the Rastaban system unable to leave. I've had the popup about not being able to leave the system come up in one of the pirate systems multiple times (I can't remember which one somewhere around Kappa Centauri) but I always launched anyway and was able to leave. This time ar Rastaban no such luck. Can't plot a course as no routes out of system and can't do anything in the system but sit around and wait. I am running whatever the latest version is as of my download two days ago.

**Comment by DalynWoodard** [comment]
OK I loaded up the game on my other computer (the  one I play on) and I'm running version 0.9.13 of endless sky. I went in and checked and sure enough I no longer had a hyperdrive installed. I am still not sure how that happened, but not really worried about it. I bought one and now I can play as usual again. Thanks for the help.

**Comment by Zitchas** [comment]
It seems weird to have lost a hyperdrive for no reason, but I'm glad we could help trouble shoot it.

**Comment by DalynWoodard** [comment]
LOL I think  I figured out how it happened. I'm a bit embarrassed that I didn't notice at the time. I've been mainly making my money by going to pirate systems and disabling and looting the ships that attack me then returning to a friendly planet to sell the loot. I suspect I accidently double clicked on sell item when I had one looted hyperdrive in hold and sold the one I had installed without noticing because I was also watching the college baseball national championship region game. :)
